modules/raft/src/main/java/org/apache/ignite/internal/raft/RaftGroupServiceImpl.java

```diff
@@ -575,8 +575,12 @@ private void handleThrowable(
     ) {
         if (recoverable(err)) {
             LOG.warn(
-                    "Recoverable error during the request type={} occurred (will be retried on the randomly selected node): ",
-                    err, sentRequest.getClass().getSimpleName()
+                    "Recoverable error during the request occurred (will be retried on the randomly selected node) "
+                            + "[type={}, groupId={}, peer={}].",
+                    err,
+                    sentRequest.getClass().getSimpleName(),
```

Comment:
I would suggest to log the request itself.

Reply:
Agree. And because request contains not only a command but alsoa groupId it won't be necessary to log latter explicitly.
```
    @IgniteToStringInclude
    private final WriteCommand command;

    @IgniteToStringInclude
    private final String groupId;
```